scripts/pull_github_pr.sh

```diff
@@ -70,10 +78,15 @@ check_jenkins_job_status() {
     ORANGE='\033[0;33m'
     NC='\033[0m'
     echo -e "${ORANGE}\nWARNING:${NC} $jenkins_job is not stable"
-    exit 1
+    echo "exiting script since $lastCompletedBuild is unstable, you can force it by running ./script/pull_github_pr.sh <pr num> --force"
```

Comment:
I think you shouldn't print this message if "--force" is already active. Perhaps print a different message saying the build is unstable but don't say "exiting" or recommend using "--force".